packages/keyring-eth-hd/src/hd-keyring.ts

```diff
@@ -253,7 +248,7 @@ export class HdKeyring {
       privateKey instanceof Uint8Array,
       'Expected private key to be of type Uint8Array',
     );
-    return bytesToHex(privateKey);
+    return remove0x(bytesToHex(privateKey));
```

Comment:
Is this expected to be a breaking change?

Reply:
No as we were previously using `bytesToHex` from `cryptography/utils` which doesn't include the `0x` prefix, so this just aligns with that but using the `@metamask/utils` version for consistency and performance.